include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -20,6 +20,47 @@
 namespace blmc_robots
 {
 
+// TODO what is the best scope for those homing-related types?
+
+/**
+ * @brief Possible return values of the homing
+ */
+enum HomingReturnCode {
```

Comment:
could we make this an enum class instead of a plain enum? this encapsulates things a bit nicer.

Reply:
Done. I was working too much with plain C recently, so I completely forgot about this distinction in C++...

Reply:
Thanks!